src/main/java/com/sda/eventapp/configuration/SecurityConfig.java

```diff
@@ -20,7 +20,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .authorizeHttpRequests(requests -> requests
                         .requestMatchers("/", "/home",
                                 "/user/register",
-                                "/detail-view/**",
+                                "/detail-view/*",
```

Comment:
Single asterisk is wildcard for part of the link, up to the '/' sign.
Double asterisk is a wildcard for whole link.

Thanks @degorskiprzemyslaw for this explanation!